core/unit_tests.c

```diff
@@ -67,22 +67,22 @@ void
 write_ymm_aux(dr_ymm_t *buffer, int regno)
 {
     switch (regno) {
-#    define MOVE_TO_YMM_VEX(B, N)   \
-        asm volatile("vmovdqu %0, " \
-                     "%%ymm" #N     \
-                     :              \
-                     : "m"(B[N]))
-#    define MOVE_TO_YMM_EVEX(B, N)    \
-        asm volatile("vmovdqu32 %0, " \
-                     "%%ymm" #N       \
+#    define MOVE_TO_YMM_VEX(BUF, NUM) \
```

Comment:
style violation: lower-case for parameter names

Ditto below

Reply:
Fixed.